src/nerve/frontends/cli/server/graph.py

```
@@ -82,33 +80,33 @@ async def run_graph_file(
     from nerve.server.protocols import Command, CommandType
     from nerve.transport import UnixSocketClient
 
-    print(f"Loading: {filepath}")
+    click.echo(f"Loading: {filepath}")
 
     try:
         graph_def = load_graph_from_file(filepath)
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)